recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/cython/cython/archive/69ad775b0a852dd38a8529eb2fe4e562d5fb2e75.tar.gz
-  sha256: 82fd9679b4d335175335651a1105fd4a99078e5dc05bf8022e6e21ca69579eb0
+  url: https://files.pythonhosted.org/packages/f5/6a/b820fd8626fa6a658880815dc99dea8318438dabee96d75e261d050d94d8/cython-3.1.0b1.tar.gz
```

Comment:
Let's please use a url that's more canonical - i.e. from the GitHub tarball or something published on PyPI (even if the pypi link perhaps resolves to the link above behind the scenes).

Reply:
My preference would be
`https://github.com/cython/cython/archive/refs/tags/{{ version }}.tar.gz` like on main

Reply:
Sure - didn't think about it, just grabbed the actual PyPI artifact link. Mind making the change? I already signed off for the day.